src/drt/src/dr/FlexDR.cpp

```diff
@@ -1823,6 +1824,43 @@ void FlexDR::searchRepair(const SearchRepairArgs& args)
                   199,
                   "  Number of violations = {}.",
                   getDesign()->getTopBlock()->getNumMarkers());
+    if (getDesign()->getTopBlock()->getNumMarkers() > 0) {
+      // report violations
+      std::map<std::string, std::map<frLayerNum, uint>> violations;
+      std::set<frLayerNum> layers;
+      const std::map<std::string, std::string> relabel
+          = {{"Lef58SpacingEndOfLine", "EOL"},
+             {"Lef58CutSpacingTable", "CutSpcTbl"},
+             {"Lef58EolKeepOut", "eolKeepOut"}};
+      for (const auto& marker : getDesign()->getTopBlock()->getMarkers()) {
```

Comment:
I don't think users will care about recheck violations as they are really internal to the router.  I would limit them to a developer debug level.

Reply:
Wouldn't it be wierd that the number of markers reported by info message "199" doesn't match the violations reported here if we remove the recheck violations? Or should we remove the recheck violations from the info message as well?

Reply:
That's a good point.  Removing them from 199 could make it appear that there are zero violations and yet the router keeps running which will seem odd too.  I guess we can just leave it the way it is and see if people are confused or not.